packages/pwa/app/hooks/use-categories.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {useContext} from 'react'
+import {CategoriesContext} from '../contexts'
+
+/**
+ * Custom React hook to get the categories
+ * @returns {{currency: string, setCurrency: function}[]}
```

Comment:
Oh looks like this is a typo due to copy & paste..

```suggestion
 * @returns {{categories: Object, setCategories: function}[]}
```